cmake/package_helpers.cmake

```diff
@@ -94,10 +117,19 @@ macro(ginkgo_add_external_target new_target external_name includedir libdir buil
         set_target_properties(${new_target} PROPERTIES IMPORTED_LOCATION_RELEASE ${${external_name}_LIBRARY_RELEASE})
         set_target_properties(${new_target} PROPERTIES IMPORTED_LOCATION_DEBUG ${${external_name}_LIBRARY_DEBUG})
         # Since we do not really manage other build types, let's globally use the DEBUG symbols
-        if (NOT CMAKE_BUILD_TYPE MATCHES "[Rr][Ee][Ll][Ee][Aa][Ss][Ee]"
-            AND NOT CMAKE_BUILD_TYPE MATCHES "[Dd][Ee][Bb][Uu][Gg]")
+        if(MSVC)
+        # Only Debug build uses MDd or MTd, and others use MD or MT.
+        # MSVC would like to use same runtime library, so we use Debug third-party in Debug and Release third-party in Release.
+            set_target_properties(${new_target} PROPERTIES IMPORTED_LOCATION_DEBUG
+                    ${${external_name}_LIBRARY_DEBUG})
```

Comment:
I am maybe wrong, but this is the same as line 118, no? Why do you need this line?

Reply:
you are right. it is a duplication